{{cookiecutter.project_slug}}/README.md

```diff
@@ -0,0 +1,104 @@
+[![Conda-Forge][conda-badge]][conda-link]
+[![PyPI platforms][pypi-platforms]][pypi-link]
+
+[![GitHub Discussion][github-discussions-badge]][github-discussions-link]
+[![Gitter][gitter-badge]][gitter-link]
+
+<!-- prettier-ignore-start -->
+[actions-badge]:            {{cookiecutter.url}}/workflows/CI/badge.svg
+[actions-link]:             {{cookiecutter.url}}/actions
+[conda-badge]:              https://img.shields.io/conda/vn/conda-forge/{{cookiecutter.project_name}}
+[conda-link]:               https://github.com/conda-forge/{{cookiecutter.project_name}}-feedstock
+[github-discussions-badge]: https://img.shields.io/static/v1?label=Discussions&message=Ask&color=blue&logo=github
+[github-discussions-link]:  {{cookiecutter.url}}/discussions
+[gitter-badge]:             https://badges.gitter.im/{{cookiecutter.url}}/community.svg
+[gitter-link]:              https://gitter.im/{{cookiecutter.url}}/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+[pypi-link]:                https://pypi.org/project/{{cookiecutter.project_name}}/
+[pypi-platforms]:           https://img.shields.io/pypi/pyversions/{{cookiecutter.project_name}}
+[pypi-version]:             https://img.shields.io/pypi/v/{{cookiecutter.project_name}}
+[rtd-badge]:                https://readthedocs.org/projects/{{cookiecutter.project_name}}/badge/?version=latest
+[rtd-link]:                 https://{{cookiecutter.project_name}}.readthedocs.io/en/latest/?badge=latest
+
+<!-- prettier-ignore-end -->
+
+{{cookiecutter.short_description}}
+
+This project is developed in collaboration with the [Centre for Advanced Research Computing](https://ucl.ac.uk/arc), University College London.
+
+## About
+
+{{cookiecutter.long_description}}
+
+### Project Team
+{{cookiecutter.fullname}}  ([{{cookiecutter.email}}](mailto:{{cookiecutter.email}}))
+<!-- TODO: how do we have an array of collaborators ? -->
+
+### Research Software Engineering Contact
+
+Centre for Advanced Research Computing, University College London
+  ([arc-collab@ucl.ac.uk](mailto:arc-collab@ucl.ac.uk))
+
+## Built With
+
+<!-- TODO: can cookiecutter make a list of frameworks? -->
+[Framework 1](https://something.com)  
+[Framework 2](https://something.com)  
+[Framework 3](https://something.com)  
+
+## Getting Started
+
+### Prerequisites
+
+Any tools or versions of languages needed to run code. For example specific Python or Node versions. Minimum hardware requirements also go here.
+
+### Installation
+
+How to build or install the application.
+
+### Running Locally
+
+How to run the application on your local system.
+
+### Running Tests
+
+How to run tests on your local system.
+
+## Roadmap
+
+- [x] Initial Research  
+- [ ] Minimum viable product <-- You are Here  
+- [ ] Alpha Release  
+- [ ] Feature-Complete Release  
+
+## License
+
+{{cookiecutter.license}}
+
+## Citation
+
+Please cite [xx.yyy/zenodo.zzzz](https://doi.org/xx.yyy/zenodo.zzzzz) for this work if you use this code.

```

Comment:
this be in citation.cif instead, but let's do that later on

Reply:
`CITATION.cff` ^